staging/src/k8s.io/code-generator/cmd/go-to-protobuf/protoc-gen-gogo/main.go

```diff
@@ -28,5 +30,18 @@ import (
 )
 
 func main() {
-	command.Write(command.Generate(command.Read()))
+	// read input
+	request := command.Read()
+
+	// if we're given paths as inputs, generate .pb.go files based on those paths
+	for _, file := range request.FileToGenerate {
+		if strings.Contains(file, "/") {
```

Comment:
Contains or prefix? `strings.HasPrefix(file, "/")`

Reply:
contains. when given paths, it looks like `k8s.io/kubernetes/vendor/k8s.io/api/core/v1/generated.proto`

Reply:
should we put in the comment, an example of these expected paths to make it easier for other people looking at this code ?